.circleci/config.yml

```diff
@@ -266,17 +260,17 @@ jobs:
 
   docs_test:
     docker:
-      - image: python:3.10.5-bullseye@sha256:dac61c6d3e7ac6deb2926dd96d38090dcba0cb1cf9196ccc5740f25ebe449f50
+      - image: cimg/python:3.11.2
     steps:
       - checkout
       - restore_cache:
-          name: Restoring Python docs dependency cache
-          key: v1-requirements-{{ checksum "docs/requirements.txt" }}
+          name: Restoring docs dependency cache
+          key: v2-docs-requirements-{{ checksum "poetry.lock" }}
```

Comment:
This will unfortunately bust the cache more than just the checksum of `docs/requirements.txt`, but hopefully this process will still go quickly since we only have 2 dependencies in the docs group.